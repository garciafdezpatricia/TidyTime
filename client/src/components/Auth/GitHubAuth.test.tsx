import { render } from "@testing-library/react";
import '@testing-library/jest-dom';
import GitHubAuthButton from '@/src/components/Auth/GitHubAuth';

describe('GitHubAuthButton', () => {
    it('renders without crashing', () => {
      const { container } = render(<GitHubAuthButton />);
      expect(container).toBeInTheDocument();
    });

    it('matches snapshot', () => {
        const { asFragment } = render(<GitHubAuthButton />);
        expect(asFragment()).toMatchSnapshot();
      });
  });