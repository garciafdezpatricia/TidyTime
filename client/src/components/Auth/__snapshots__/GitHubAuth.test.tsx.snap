// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubAuthButton matches snapshot 1`] = `
<DocumentFragment>
  <button
    class="auth-button"
  >
    <img
      alt=""
      class="btn-icon"
      src="./github.svg"
    />
    list.github
  </button>
</DocumentFragment>
`;
